axelar-chains-config/info/testnet.json

```diff
@@ -2453,40 +2318,233 @@
+            "GasCollected": "0x021b9da78117b4fac344200f82b4c5d581c8eaffb6ee0197073d4e46e515b241::events::GasCollected",
+            "GasService_v0": "0x021b9da78117b4fac344200f82b4c5d581c8eaffb6ee0197073d4e46e515b241::gas_service_v0::GasService_v0",
+            "GasService": "0x021b9da78117b4fac344200f82b4c5d581c8eaffb6ee0197073d4e46e515b241::gas_service::GasService",
+            "GasCollectorCap": "0x021b9da78117b4fac344200f82b4c5d581c8eaffb6ee0197073d4e46e515b241::gas_service::GasCollectorCap"
+          },
+          "objects": {
+            "GasCollectorCap": "0x47f89dc0f25de4da21278ef45187be507f7f40b1ec90745a351ff198747b6398",
+            "GasService": "0xd934b94c0ee1b741c7ef6730d4b014d4618b281ca5e641ac8d133b7234b4d6a9",
+            "GasServicev0": "0x737706f16c72f9110916f266714967e518868b4dbf08774c1a487b4ccc2971e8"
+          }
+        },
+        "Abi": {
+          "address": "0x27c00fa7a826b8eb6219b13228ddf35a95dd4ded91bafef340d96c851b872896",
+          "versions": [
+            "0x27c00fa7a826b8eb6219b13228ddf35a95dd4ded91bafef340d96c851b872896"
+          ],
+          "structs": {
+            "AbiReader": "0x27c00fa7a826b8eb6219b13228ddf35a95dd4ded91bafef340d96c851b872896::abi::AbiReader",
+            "AbiWriter": "0x27c00fa7a826b8eb6219b13228ddf35a95dd4ded91bafef340d96c851b872896::abi::AbiWriter"
+          }
+        },
+        "Operators": {
+          "address": "0xde375ddbd37f64e7952da48ebe6be4888abd6c4f6a0958512465006e35fdc6a6",
+          "versions": [
+            "0xde375ddbd37f64e7952da48ebe6be4888abd6c4f6a0958512465006e35fdc6a6"
+          ],
+          "structs": {
+            "OwnerCap": "0xde375ddbd37f64e7952da48ebe6be4888abd6c4f6a0958512465006e35fdc6a6::operators::OwnerCap",
+            "OperatorCap": "0xde375ddbd37f64e7952da48ebe6be4888abd6c4f6a0958512465006e35fdc6a6::operators::OperatorCap",
+            "Operators": "0xde375ddbd37f64e7952da48ebe6be4888abd6c4f6a0958512465006e35fdc6a6::operators::Operators",
+            "Borrow": "0xde375ddbd37f64e7952da48ebe6be4888abd6c4f6a0958512465006e35fdc6a6::operators::Borrow",
+            "OperatorAdded": "0xde375ddbd37f64e7952da48ebe6be4888abd6c4f6a0958512465006e35fdc6a6::operators::OperatorAdded",
+            "OperatorRemoved": "0xde375ddbd37f64e7952da48ebe6be4888abd6c4f6a0958512465006e35fdc6a6::operators::OperatorRemoved",
+            "CapabilityStored": "0xde375ddbd37f64e7952da48ebe6be4888abd6c4f6a0958512465006e35fdc6a6::operators::CapabilityStored",
+            "CapabilityRemoved": "0xde375ddbd37f64e7952da48ebe6be4888abd6c4f6a0958512465006e35fdc6a6::operators::CapabilityRemoved"
+          },
+          "objects": {
+            "Operators": "0xef688fe7460ae9da3b45af8fb4219a6803d16c2049498613c2043a1996d6eda0",
+            "OwnerCap": "0x7d71d070eeea0eeaca7a44f52747b2e9942c9d3fa921093ed5e5abd7bec5562c"
+          }
+        },
+        "ITS": {
```

Comment:
Why was ITS deployed? ITS release hasn't been created yet. Shouldn't  include invalid contracts in the config